src/applications/personalization/dashboard/reducers/prescriptions.js

```diff
@@ -69,7 +68,7 @@ export default function prescriptions(state = initialState, action) {
         };
       }
 
-      return assign(state, newState);
+      return Object.assign({}, state, newState);
```

Comment:
might as well just do `{...state, ...newState}`?

Reply:
Yeah I agree, this was ported over from a larger PR that got split.